src/transaction/posting.rs

```diff
@@ -93,6 +103,55 @@ pub enum PriceType {
     Total,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct LotAttributes<'a> {
```

Comment:
keep it private for now,

```suggestion
struct LotAttributes<'a> {
```